AppInsightsCommon/src/Interfaces/IEventTelemetry.ts

```diff
@@ -8,4 +8,14 @@ export interface IEventTelemetry {
      * @memberof IEventTelemetry
      */
     name: string;
+
+    /**
+     * Property bag to contain additional custom properties (Part C)
+     */
+    properties?: { [key: string]: string };
```

Comment:
can type be switched to object?

Reply:
not in this PR. I need to follow up on two things before I can make the switch. Channel will still need to downcast till there is better support for object.